cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/services/ProgramRunStatusMonitorService.java

```diff
@@ -145,4 +157,38 @@ protected long runTask() {
       return true;
     };
   }
+
+  private void emitForceTerminatedRunsMetrics(ProgramRunId programRunId, RunRecordDetail recordedRunRecord,
+                                              String metricName) {
+    Optional<ProfileId> profile = SystemArguments.getProfileIdFromArgs(programRunId.getNamespaceId(),
+                                                                       recordedRunRecord.getSystemArgs());
+    Map<String, String> additionalTags = getAdditionalTagsForProgramMetrics(recordedRunRecord);
+    profile.ifPresent(profileId -> emitProfileMetrics(programRunId, profileId, metricName, additionalTags));
+  }
+
+  private Map<String, String> getAdditionalTagsForProgramMetrics(RunRecordDetail runRecord) {
+    Map<String, String> additionalTags = new HashMap<>();
+    // don't want to add the tag if it is not present otherwise it will result in NPE
+    additionalTags.computeIfAbsent(Constants.Metrics.Tag.PROVISIONER,
+                                   provisioner -> SystemArguments.getProfileProvisioner(runRecord.getSystemArgs()));
+    additionalTags.computeIfAbsent(Constants.Metrics.Tag.CLUSTER_STATUS,
```

Comment:
do we need this tag? Don't think we would ever query for killed runs by cluster status?